src/styles/jolocom-theme.ts

```diff
@@ -1,127 +1,164 @@
-    fontSize: 13,
-    fontStyle: 'normal',
-    fontWeight: '300',
-    color: palette.textColor_grey
+  light: {
+    labelDisplayField: {
+      fontSize: 16,
+      color: palette.primaryColor
+    },
+    textDisplayField: {
+      fontSize: 20,
+      color: palette.primaryColorBlack
+    }
   },
-  labelButton: {
-    fontSize: '14px',
-    fontStyle: 'normal',
-    fontWeight: '500',
-    color: palette.accent1Color,
-    textTransform: 'uppercase'
+  // old styles TODO: remove when finished with refactor ui
+  headline: {
+     fontSize: 24,
+     fontStyle: 'normal',
+     fontWeight: '200',
+     color: palette.textColor_grey
   },
-  screenHeader: {
-    fontSize: '20px',
-    fontStyle: 'normal',
-    fontWeight: '500',
-    color: palette.textColor
+   subheadline: {
+     fontSize: 20,
+     fontStyle: 'normal',
+     fontWeight: '200',
+     color: palette.textColor_grey
   },
-  userName: {
-    fontSize: '28px',
-    fontStyle: 'normal',
-    fontWeight: '400',
-    color: palette.textColor
+   sectionheader: {
+     fontSize: 20,
+     fontStyle: 'normal',
+     fontWeight: '400',
+     color: palette.textColor
+   },
+   labelInputFields: {
+     fontSize: 13,
+     fontStyle: 'normal',
+     fontWeight: '300',
+     color: palette.textColor_darkBrown75
+   },
+   inputFields: {
+     fontSize: 16,
+     fontStyle: 'normal',
+     fontWeight: '400',
+     color: palette.textColor_silverGrey
+   },
+   contentInputFields: {
+     fontSize: 16,
+     fontStyle: 'normal',
+     fontWeight: '400',
+     color: palette.textColor
+   },
+   textCopy: {
+     fontSize: 13,
+     fontStyle: 'normal',
+     fontWeight: '300',
+     color: palette.textColor_grey
+   },
+   labelButton: {
+     fontSize: '14px',
+     fontStyle: 'normal',
+     fontWeight: '500',
+     color: palette.accent1Color,
+     textTransform: 'uppercase'
+   },
+   screenHeader: {
+     fontSize: '20px',
+     fontStyle: 'normal',
+     fontWeight: '500',
+     color: palette.textColor
+   },
+   userName: {
+     fontSize: '28px',
+     fontStyle: 'normal',
+     fontWeight: '400',
+     color: palette.textColor
   }
 }
 
 export const JolocomTheme = {
-
-  primaryColorBlack: 'black',
-  primaryColorSand: '#ffefdf',
-  primaryColorGrey: '#fafafa',
-  primaryColorWhite: 'white',
-
-  labelFontSize: 20,
-  headerFontSize: 22,
-
-
   contentFontFamily: 'Roboto, sans-serif',
   palette,
   textStyles,
-  appBar: {
-    color: '#f8f9fb',
-    textColor: '#4b132b'
-  },
-  // Used for actionable modals, like `add node`, `link node`
-  actionAppBar: {
-    color: palette.primary1Color,
-    textColor: '#ffffff'
-  },
-  inkBar: {
-    backgroundColor: '#b3c90f'
-  },
-  tabs: {
-    backgroundColor: '#f8f9fb',
-    selectedTextColor: '#4b132b',
-    textColor: '#4b132b'
-  },
-  textField: {
-    focusColor: lime['700']
-  },
-  jolocom: {
-    gray1: '#9ba0aa',
-    gray2: '#c3c6cc',
-    gray3: '#e1e2e5',
-    gray4: '#f8f9fb',
-    gray5: '#f7f7f7'
+  button: {
```

Comment:
I would extract it to a separate const too, so that `JolocomTheme` would be more readable